### ISSUE TITLE
Add Gasoline to Fuels?

### ISSUE THREAD

**Issue by PessimisticBastard** [bdew-minecraft/generators#150]
Could you add Gasoline from Immersive Petroleum to the list of fuels the gas turbine can use? I always have so much of the stuff left around not doing much of anything.

**Comment by oxinabox** [comment]
I was just thinking the same thing.
I think one can add it ones self though, via [config](https://bdew.net/generators/configuration/).
When I work that out, I will post details

update:
I think all you have to do is add a file with what ever name you want, with the `.cfg` extension (I used `iepetrol.cfg`)
and give it the contents:

```
recipes {
    turbine-fuel: fluid_gasoline 5.12 MJ/mB
}
```

Those figures are based on 10RF->1MJ,
and the IE:Petrolium portable generator using 5mb/tick, to output 265RF/t (that is from their source file/config)

But that doesn't seem to work.
So I must be doing something wrong.

**Comment by PessimisticBastard** [comment]
I ended up adding it directly to the config file in the .jar itself under the fuels section.

Also, gasoline is just called gasoline in the fluids section: 

I.E. it'd be 

turbine-fuel: gasoline 60 MJ/mB 

That's how I did it. (Yes, that's significantly higher than it is in a portable generator, but it annoys me that gasoline makes so much less RF than diesel, when the IRL energy density of gasoline (46.4 MJ/kg) vs diesel (48 MJ/kg) isn't THAT big a difference. Even taking in the efficiency of a diesel turbine (usually around 30% efficient) vs your usual portable generator (between 18-20%), it still doesn't make sense.)
